L/libjulia/common.jl

```diff
@@ -5,7 +5,8 @@ using BinaryBuilder, Pkg
 include("../../fancy_toys.jl") # for get_addable_spec and should_build_platform
 
 # list of supported Julia versions
-julia_full_versions = [v"1.6.3", v"1.7.0", v"1.8.2", v"1.9.0", v"1.10.0", v"1.11.1", v"1.12.0-rc1", v"1.13.0-DEV"]
+julia_full_versions = [v"1.10.0", v"1.11.1", v"1.12.0-rc1", v"1.13.0-DEV"]
+libjulia_minimal_supported_julia_version = Base.thispatch(minimum(julia_versions))
```

Comment:
* `julia_versions` is used here before it is defined.
* I would propose the name `libjulia_julia_compat`

Reply:
both done